src/math/p_div.c

```diff
@@ -29,3 +29,33 @@ void p_div_f32(const float *a, const float *b, float *c,
         *(c + i) = *(a + i) / *(b + i);
     }
 }
+
+/**
+ *
+ * Element wise vector division between input vector 'a' and scalar 'b'
+ *
+ * @param a     Pointer to input vector
+ *
+ * @param b     Pointer to input scalar
+ *
+ * @param c     Pointer to output vector
+ *
+ * @param n     Size of 'a' and 'c' vector.
+ *
+ * @param p     Number of processor to use (task parallelism)
+ *
+ * @param team  Team to work with 
+ *
+ * @return      None
+ *
+ */
+
+void p_div_s(const float *a, const float *b, float *c,
+               int n, int p, p_team_t team)
+{
+
+    int i;
+    for (i = 0; i < n; i++) {
+        *(c + i) = *(a + i) / *(b);
```

Comment:
Nope, I haven't.  But I was thinking a little bit more generic ( ARM and Intel platforms which are also targeted by the library in the future). Furthermore I think having it this way makes the code more clear, if someone wants to speed up division by using this trick, he can do so as an active choice ( and then think about the numeric stability issues that might arise in his code), and call multiplication explicitly .